src/frontend/components/Chat/SharedChatComponents/ChatMessageItem/index.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { DateTime } from 'luxon';
+import React from 'react';
+import { renderImageSnapshot } from 'utils/tests/imageSnapshot';
+import { screen } from '@testing-library/react';
+
+import { ChatMessageItem } from '.';
+
+describe('<ChatMessageItem />', () => {
+  it('renders the chatMessagesMetadatas and checks displayed information and sizes are correct.', () => {
+    renderImageSnapshot(
+      <ChatMessageItem
+        msg={{
+          sentAt: DateTime.fromISO('2020-01-01T12:12:12'),
+          sender: 'John Doe',
+          content: 'This is an exemple message.',
```

Comment:
```suggestion
          content: 'This is an example message.',
```

Reply:
correct in commit [f6f606](https://github.com/openfun/marsha/pull/1247/commits/f6f606a391fbf5951e5cb59b9227949d9467522e)